packages/server/lib/controllers/apiAuth.controller.ts

```diff
@@ -134,6 +139,36 @@ class ApiAuthController {
 
             const { apiKey } = req.body;
 
+            const credentials: ApiKeyCredentials = {
+                type: AuthModes.ApiKey,
+                apiKey
+            };
+
+            const connectionResponse = await connectionTestHook(
+                config?.provider,
+                template,
+                credentials,
+                connectionId,
+                providerConfigKey,
+                environmentId,
+                connectionConfig,
+                tracer
+            );
+
+            if (!isOk(connectionResponse)) {
```

Comment:
any reason to use negation here instead of `isErr`?

Reply:
Nope, updated with https://github.com/NangoHQ/nango/pull/1656/commits/3dd00af870d00d0e326e46eaae13e1e8fed622e8